backend/app_logging/example.py

```
@@ -0,0 +1,268 @@
+    print("  POST /api/demo/database         - Database operation logging")
+    print("  GET  /api/demo/slow             - Performance monitoring demo")
+    print("  GET  /api/demo/error?type=X     - Error handling demo")
+    print("  POST /api/demo/login            - Authentication logging")
+    print("  POST /api/demo/context          - Rich context logging")
+    print("\nExample requests:")
+    print("  curl http://localhost:5000/api/demo/basic")
+    print("  curl -X POST http://localhost:5000/api/demo/login -H 'Content-Type: application/json' -d '{\"username\":\"demo\"}'")
+    print("  curl http://localhost:5000/api/demo/error?type=validation")
+    print("="*60)
+
+    app.run(debug=True, port=5000)
```

Ruff (0.13.1)

Use of `debug=True` in Flask app detected

(S201)